index.js

```diff
@@ -8,14 +8,20 @@ const io = require('socket.io')(server)
 
 app.use('/', express.static('client'))
 
+// 
 io.on('connection', socket => {
-    console.log('a user connected');
+    // creating a test-room for development
+    // appends to a Set object on socket
+    // socket.rooms 
+    socket.join('test room')
 
+    // here we should instantiate a game obj
+    // to track game state and stack history
 
-    socket.on('gameplay-stroke', data=> {
-		console.log(data)
-        io.emit('response', {
-            'response': 'ack'
-        })
+    // update users in room for new image
+    socket.on('gameplay-stroke', data => {
+        // get which room from sender
+        // send to all in same room
+        io.in([...socket.rooms][1]).emit('new-image', data)
```

Comment:
is this the cleanest it can be ? `...socket.rooms][1]`

I was thinking the socket api would let you do:
```
io.in('test room').forEachEmit('new-image', data)
```